### PR TITLE
Add redelivery object for narrowcast

### LINKED ISSUE #305: Support Redelivery object for narrowcast messages
News: https://developers.line.biz/en/news/2020/09/15/messaging-api-narrowcast-requestId/
Docs: [Redelivery object](https://developers.line.biz/en/docs/messaging-api/sending-messages/#redelivery-object)

### PR DESCRIPTION
**Pull request by hinoguma**
fixes #305 
- [ ] [Support redelivery object for narrowcast](https://developers.line.biz/en/reference/messaging-api/#narrowcast-recipient)

### CODE REVIEW

File: tests/LINEBot/NarrowCastTest.php [hunk 1]
```diff
@@ -85,6 +86,13 @@ public function testSendNarrowcast()
                         ->setNot(
                             AudienceRecipientBuilder::builder()
                                 ->setAudienceGroupId(21234567890)
+                        ),
+                    RedeliveryRecipientBuilder::builder()
```
Comment (Vaduz) [inline tests/LINEBot/NarrowCastTest.php:90]: Please add assertion/tests in `$testRunner` to verify the data you appended.

**Review by Vaduz — CHANGES_REQUESTED** [review]
The new class looks good but you are missing tests.

File: tests/LINEBot/NarrowCastTest.php (hunk 1 above)
Comment (hinoguma) [reply tests/LINEBot/NarrowCastTest.php:101]: @Vaduz thank you for reviewing. added test assert.
https://github.com/line/line-bot-sdk-php/pull/306/commits/daebefce4e04f78cc329a5b0bef6997852cf9668

**Review by moririnson — APPROVED** [review]

**Review by Vaduz — APPROVED** [review]

**Comment by moririnson** [comment]
thanks for your contribution!
i'll merge this.

**Pull request merged**